settings/LoanPolicyForm.js

```diff
@@ -114,6 +116,10 @@ class LoanPolicyForm extends React.Component {
     if (altRenewPeriod && altRenewPeriod.intervalId && !altRenewPeriod.duration) {
       this.props.change('renewalsPolicy.period.duration', 1);
     }
+    const openingTimeOffset = (allValues.loansPolicy && allValues.loansPolicy.openingTimeOffset);
+    if (openingTimeOffset && openingTimeOffset.intervalId && !openingTimeOffset.duration) {
```

Comment:
Move condition inside if to the descriptive variable and add an empty line before and after `const`.